packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1223,10 +1223,6 @@ class WidgetsApp extends StatefulWidget {
     SingleActivator(LogicalKeyboardKey.tab, shift: true): PreviousFocusIntent(),
 
     // Scrolling
-    SingleActivator(LogicalKeyboardKey.arrowUp): ScrollIntent(direction: AxisDirection.up),
```

Comment:
These shortcuts was ignored because the shortcut is built on top of the DefaultTextEditingShortcuts which prevented these shortcut from bubbling up.

I am not sure if this is the correct behavior, but there are tests to verify the arrow key does not cause scroll in CupertinoTextField, so I remove these shortcuts to maintain the same behavior

Reply:
Maybe @Piinks knows?  These shortcuts allow scrolling on a normal Scrollable, and I think they were not used by text inputs because the arrow keys move the cursor instead.